jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/HoverTests.java

```diff
@@ -17,70 +17,116 @@
-                    .cssClasses(Matchers.hasItem(not("show-btns"))));
+            card.getPlayerButtons().stream()
+                    .map(Button::has)
+                    .forEach(textAssert -> textAssert.cssClasses(Matchers.hasItem(not("show-btns"))));
 
             card.getHover().show();
-            card.getPlayerButtons().stream().map(Button::has).forEach(textAssert -> textAssert.cssClass("show-btns"));
+            card.getPlayerButtons().stream()
+                    .map(Button::has)
+                    .forEach(textAssert -> textAssert.cssClass("show-btns"));
+
+            card.getPlayerButtons().forEach(button -> button.core().isClickable());
             card.getPlayerButtons().forEach(HasClick::click);
 
             card.getHover().suppress();
-            card.getPlayerButtons().stream().map(Button::has).forEach(textAssert -> textAssert
-                    .cssClasses(Matchers.hasItem(not("show-btns"))));
+            card.getPlayerButtons().stream()
+                    .map(Button::has)
+                    .forEach(textAssert -> textAssert.cssClasses(Matchers.hasItem(not("show-btns"))));
         });
     }
 
-    @Test
-    public void cookingCardTest() {
-        cookingHeader.has().text(containsString("cooking utensils"));
-        cartButton.click();
+    @Test(description = "Assert that Music Card list has proper texts")
+    public void musicCardListTextsTest() {
+        SUBHEADINGS.forEach(subheading -> jdiAssert(musicHoverCards.stream()
+                .map(MusicHoverCard::getSubheading)
+                .map(IsText::getText)
+                .collect(Collectors.toList()), Matchers.hasItem(subheading)));
+    }
+
+    @Test(description = "Assert that Cooking Card price text appears only when card is being hovered")
+    public void cookingCardPriceTextAppearsWhenHoveredTest() {
+        cookingHoverCard.priceHover().is().notVisible();
+        cookingHoverCard.priceHover().show();
+        cookingHoverCard.priceHover().has().text(containsString(COOKING_CARD_PRICE));
+        cookingHoverCard.priceHover().suppress();
+        cookingHoverCard.priceHover().is().notVisible();
+    }
+
+    @Test(description = "Assert that Cooking Card header contains proper text")
+    public void cookingCardHeaderTextTest() {
+        cookingHoverCard.cookingHeader().has().text(containsString(COOKING_CARD_HEADER_TEXT));
+    }
 
-        priceHover.show();
-        priceHover.has().text(containsString("$14.99"));
-        priceHover.suppress();
+    @Test(description = "Assert that Cooking Card button is clickable, and price text appears when clicking")
+    public void cookingCardCardButtonTest() {
+        cookingHoverCard.cartButton().core().isClickable();
+        cookingHoverCard.cartButton().click();
+        cookingHoverCard.priceHover().has().text(containsString(COOKING_CARD_PRICE));
+        cookingHoverCard.priceHover().suppress();
     }
```

Comment:
Sorry, did not notice at previous review
1) what feature of 'hover' do you check here?
2) as you may see in the code below 'click' does not open the priceWindow and it opens only because of 'hover'
But in the test it looks like 'click' opens the priceWindow. It's a logic mistake
`<template>
  <v-hover v-slot="{ hover }">
    <v-card
      class="mx-auto"
      color="grey lighten-4"
      max-width="600"
   
        <v-expand-transition>
          <div
            v-if="hover"
            class="d-flex transition-fast-in-fast-out orange darken-2 v-card--reveal text-h2 white--text"
            style="height: 100%;"
          >
            $14.99
          </div>
        </v-expand-transition>
      </v-img>
      <v-card-text
        class="pt-6"
        style="position: relative;"
      >
        <v-btn
          absolute
          color="orange"
          class="white--text"
          fab
          large
          right
          top
        >
          <v-icon>mdi-cart</v-icon>
        </v-btn>
        <div class="font-weight-light grey--text text-h6 mb-2">
          For the perfect meal
        </div>
        <h3 class="text-h4 font-weight-light orange--text mb-2">
          QW cooking utensils
        </h3>
        <div class="font-weight-light text-h6 mb-2">
          Our Vintage kitchen utensils delight any chef.<br>
          Made of bamboo by hand
        </div>
      </v-card-text>
    </v-card>
  </v-hover>
</template>`

Reply:
Thanks. It's not a feature of 'hover'... I check, if the button is clickable and if this card contains specific text. Moreover, I think. we don't need such tests:)